src/api/vfs/mod.rs

```diff
@@ -723,44 +723,70 @@ pub mod persist {
 
     #[derive(Versionize, Debug, Default)]
     struct VfsOptionsState {
-        no_open: bool,
-        no_opendir: bool,
-        no_writeback: bool,
         in_opts: u64,
         out_opts: u64,
-        killpriv_v2: bool,
         no_readdir: bool,
         seal_size: bool,
+        id_mapping_internal: u32,
+        id_mapping_external: u32,
+        id_mapping_range: u32,
```

Comment:
why introducing new fields in a clean up PR?

Reply:
It's not a clean up, but a bugfix:)